### PR TITLE
bump pylibseq

### PR DESCRIPTION
**Pull request by molpopgen**
* [ ] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x ] This PR adds a new recipe.
* [] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [X] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by molpopgen** [comment]
@bgruening It seems like something odd is happening here.  There is a complaint about svgwrite and trying to install via setuptools.  But, that doesn't seem write.  svgwrite is a dependency of msprime, which is installed in the meta.yaml and svgwrite is in conda-forge.  The recipe for msprime does show that svgwrite is a run dependency: https://github.com/conda-forge/msprime-feedstock/blob/master/recipe/meta.yaml

File: recipes/pylibseq/meta.yaml [hunk 1]
```diff
@@ -1,29 +1,31 @@
...
+  sha256: 6bfc509f90832f38e9c9ca862b8f57e35a68ba2b218a1092e3c18affef41f6dd
 
 build:
-  skip: True # [py27]
+  skip: True # [py27] 
+  skip: True # [osx]
   # If this is a new build for the same version, increment the build
   # number. If you do not include this key, it defaults to 0.
-  number: 2
+  number: 1
```
Comment (bgruening) [inline recipes/pylibseq/meta.yaml]: this should be 0

Comment (molpopgen) [reply recipes/pylibseq/meta.yaml]: Thanks. I'll change the build number.

svgwrite is a dependency of msprime, which is a listed dependency of my project.

Comment (molpopgen) [reply recipes/pylibseq/meta.yaml]: @bgruening This is the error being reported:

```
RuntimeError: Setuptools downloading is disabled in conda build. Be sure to add all dependencies in the meta.yaml  url=https://pypi.org/simple/svgwrite/
```

**Comment by bgruening** [comment]
circles everywhere :)

**Comment by molpopgen** [comment]
@bgruening @bioconda/core 
Any ideas on how to debug this?  The build system is claiming that I am attempting to install svgwrite via setuptools, which is not the case.

svgwrite is a dependency of msprime, which is in turn a dependency of pylibseq.

The relevant section of the meta.yaml for pylibseq is currently:

```
  host:
    - python
    - libsequence ==1.9.6
    - pybind11 >=2.2.3
    - msprime >=0.6.0
    - svgwrite # for msprime, but should not be needed


  run:
    - python
    - libsequence
    - msprime >=0.6.0
```

**Comment by bgruening** [comment]
@molpopgen `- svgwrite`is missing from the run section of `pylibseq` as it seems.

**Comment by molpopgen** [comment]
> @molpopgen `- svgwrite`is missing from the run section of `pylibseq` as it seems.

Do you think that will matter?  This error is occurring even if svgwrite is not listed as a build dependency (you can see previous build logs). svgwrite is being pulled in via marking msprime as a depdendency.

**Comment by bgruening** [comment]
There is something fishy not in your repo but in msprime etc, this comment does not look good ` - svgwrite # for msprime, but should not be needed` :) So I think we should fix that at first and hopefully your problem goes away with it as well.

**Comment by molpopgen** [comment]
> There is something fishy not in your repo but in msprime etc, this comment does not look good ` - svgwrite # for msprime, but should not be needed` :) So I think we should fix that at first and hopefully your problem goes away with it as well.

@bgruening OK, so if I remove that, we will be back to a failing build.  I'll ping the msprime author, who is also the maintainer of the conda-forge recipe.

**Comment by bgruening** [comment]
@molpopgen you can also create a PR against the conda-forge one if you like.

**Comment by molpopgen** [comment]
@bgruening I've been in contact with the msprime recipe maintainer, who thinks that some of the problem is due to a massive rebuild taking place on conda-forge.  I seem unable to restart failed jobs.  I tried following your instructions from [here](https://github.com/bioconda/bioconda-recipes/pull/10210#issuecomment-410089121), but I do not have permission it seems?

**Comment by bgruening** [comment]
I restarted the tests for you. You should have permissions. What do you see on top of circle-cI?

**Comment by molpopgen** [comment]
> I restarted the tests for you. You should have permissions. What do you see on top of circle-cI?

The "rerun workflow" button is grayed out and says "You need write permissions..." when I mouse over it.

When I click the "bioconda-workflow" link, I see nothing obvious about how to restart.

I am not logged in to Circle CI in any way--could that be the issue?  I've never used it outside of these PRs.

**Comment by molpopgen** [comment]
It does seem to be the logging in issue.  I just logged in via GitHub and can now see the restart button is activated.  Duh!

**Comment by molpopgen** [comment]
@bgruening This is the puzzling thing from the build log.  After building, there is another scan for dependencies.  I'm unable to track down why this is happening.  Shouldn't all dependency resolution have already happened?  In fact, you can see in the CircleCI logs that everything has been installed, including the svgwrite that is being complained about.

```
Installed /opt/conda/conda-bld/pylibseq_1543365247619/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_plac/lib/python3.5/site-packages/pylibseq-0.2.1-py3.5-linux-x86_64.egg

Processing dependencies for pylibseq==0.2.1

Searching for svgwrite

Traceback (most recent call last):

  File "setup.py", line 193, in <module>

    zip_safe=False,

  File "/opt/conda/conda-bld/pylibseq_1543365247619/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_plac/lib/python3.5/site-packages/setuptools/__init__.py", line 140, in setup
```

**Comment by molpopgen** [comment]
@jeromekelleher --Yes, I think I will try removing the build dependency.

Regarding openblas.  Conda-forge made an odd decision, IMO.  The compile GSL against openblas, but do so by deleting libgslcblas and using a symlink to openblas as a replacement. See [here](https://github.com/conda-forge/gsl-feedstock/blob/master/recipe/build.sh).  This "works" but is not how GSL recommend using an alternate blas lib.  We ran into issue with this when updating the minimum GSL in bioconda.

**Comment by jeromekelleher** [comment]
> Regarding openblas. Conda-forge made an odd decision, IMO. The compile GSL against openblas, but do so by deleting libgslcblas and using a symlink to openblas as a replacement. See here. This "works" but is not how GSL recommend using an alternate blas lib. We ran into issue with this when updating the minimum GSL in bioconda.

This is what's blocking me on the updated msprime package:
https://travis-ci.org/conda-forge/msprime-feedstock/jobs/459904966#L611

**Comment by molpopgen** [comment]
@jeromekelleher unfortunately removing msprime as a build dependency did not solve the problem

**Comment by jeromekelleher** [comment]
> @jeromekelleher unfortunately removing msprime as a build dependency did not solve the problem

<headscratch emoji> No idea what's going on here @molpopgen, sorry.

**Comment by molpopgen** [comment]
I think I may remove it as both build and run dependencies.  Doing so is
technically correct.  There is never an attempted import of msprime except
in one C++ function, and that will error out with an exception if msprime
is not installed, which is correct behavior.
__
Kevin Thornton
Associate Professor
Ecology and Evolutionary Biology
UC Irvine
www.molpopgen.org
github.com/molpopgen
github.com/ThorntonLab

**Comment by jeromekelleher** [comment]
> I think I may remove it as both build and run dependencies.  

Sounds like a plan.

**Comment by molpopgen** [comment]
Closing.  To remove msprime as a dependency, `setup.py` needs fixing, so I need to push a new version to PyPi first and try again.

**Pull request closed without merging**